packages/components/app/styles/components/pagination-bar.scss

```diff
@@ -2,4 +2,127 @@
 // PAGINATIONBAR
 //
 
-// .hds-pagination-bar {}
+@use "../mixins/focus-ring" as *;
+
+.hds-pagination-bar {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  height: 36px;
+  color: var(--token-color-foreground-primary);
+  font-size: 13px;
+}
+
+// TODO: refactor!!! (Should not reference nested component classes)
+.hds-pagination-bar__page-size {
+  > * {
+    display: flex;
+    gap: 12px; // TODO: refactor
+    align-items: center;
+  }
+
+  // Issue: attributes are spread on nested select element, not on the component's parent wrapper...
+  .hds-form-label {
+    font-weight: inherit;
+    font-size: inherit;
+  }
+
+  .hds-form-select {
+    height: 28px;
+    margin-top: 0 !important;
+    padding-top: 0;
+    padding-right: calc(0px + 24px);
+    padding-bottom: 0;
+    padding-left: 8px;
+    font-size: inherit;
+    background-position: center right 5px;
+  }
+}
+
+.hds-pagination-bar__pagination {
+  display: flex;
+  align-self: stretch;
+}
+
+.hds-pagination-bar__page-list {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  list-style-type: none;
+}
+
+.hds-pagination__page {
+  display: flex;
+}
+
+.hds-pagination__page-link {
+  @include hds-focus-ring-with-pseudo-element(
+    $top: 5px,
+    $right: 5px,
+    $bottom: 5px,
+    $left: 5px,
+  );
+  display: flex;
+  align-items: center;
+  padding: 0 12px;
+  color: inherit;
+  text-decoration: none;
+}
+
+// Hide "Previous" & "Next" labels by default (TODO: Remove from DOM using JS?)
+.hds-pagination__page-label { display: none; }
+
+// NOTE: Trying to account for white space around icons inside the links (will adjust as requested)
+.hds-pagination__page--previous .hds-pagination__page-link { padding-left: 8px; }
+.hds-pagination__page-next .hds-pagination__page-link { padding-right: 8px; }
+
+// "Compact" Pagination Very Rough WIP - TODO: Refactor
```

Comment:
Once I implement more of the actual functionality, I should be able to delete some of theses style for the "default" vs. "compact" variants of the component.